libs/wingc/src/jsify.rs

```diff
@@ -1458,7 +1458,7 @@ impl<'a> JSifier<'a> {
 
 			// TODO Hack to make sure closures match the IInflight contract from wingsdk
 			if class_type.is_closure() {
-				code.line("_hash = require('crypto').createHash('md5').update(this._toInflight()).digest('hex');")
+				code.line(format!("_id = {STDLIB_CORE}.closureId();"))
```

Comment:
perhaps resist the temptation to rename `_hash` to `_id` or move to a different commit because this pollutes all snapshots and really hard to review

Reply:
oops. I'll try to keep in mind for the future